src/main/java/uk/gov/ida/eventemitter/EventEmitterConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ida.eventemitter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.Valid;
+
+public final class EventEmitterConfiguration {
```

Comment:
What does this get used for? I can't see any usages of it?

Reply:
EventEmitterConfiguration will contain a collection of configuration items needed for configuring to connect to Amazon SQS and whether to send or not to send events to a recording system (e.g. AWS SQS's queue URL, SendToReportingSystem). Hub components will use this config for consistency.

Korny spoke to Tom and Will about how we configure the event-emitter yesterday. The preference is that each hub component has its own config, even if that means there is a lot of duplication - this means each hub component is free to be eventually split out into it’s own repo, and tested/deployed separately.  It does mean that if we use our own encryption keys, we’ll have to be careful to keep them the same.

Reply:
Maybe I need to catch up on the wider chat then - I'm still not sure that I understand what the plan is or how this class fits into it! 😄

Reply:
As discussed offline, I think we could make this a bit nicer by writing a provider (or perhaps a module).